aten/src/ATen/native/mps/operations/Distributions.mm

```diff
@@ -412,8 +412,8 @@ Tensor normal_mps(const Tensor& mean, const Tensor& std, c10::optional<Generator
   };
 
   return mps::random_mps_impl<int64_t>(result,
-                                       0.0,
-                                       1.0,
+                                       std::numeric_limits<int64_t>::min(),
```

Comment:
Changed this lower bound from 0 to -int64 min b/c more numbers -> few collisions which need to be tie broken which I think leads to a better distribution. Not sure if there are speed or portability problems here though. I will run benchmarks tomorrow.